jdaviz/configs/mosviz/plugins/viewers.py

```diff
@@ -28,23 +29,33 @@ class MosvizImageView(JdavizViewerMixin, BqplotImageView):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.init_astrowidgets_api()
         self._subscribe_to_layers_update()
 
+        self.state.show_axes = False  # Axes are wrong anyway
+        self.figure.fig_margin = {'left': 0, 'bottom': 0, 'top': 0, 'right': 0}
+
     def data(self, cls=None):
         return [layer_state.layer  # .get_object(cls=cls or self.default_class)
                 for layer_state in self.state.layers
                 if hasattr(layer_state, 'layer') and
                 isinstance(layer_state.layer, BaseData)]
 
-    def set_plot_axes(self):
-        self.figure.axes[1].tick_format = None
-        self.figure.axes[0].tick_format = None
+    def _mark_targets(self):
```

Comment:
Where does this get used, anyway? This is the only instance of the string "_mark_targets" that I see in the repository.

Reply:
I needed it to debug. I wanted to see where the RA and Dec from the table actually are. I feel like we could make it public (in one way or another) in the future but I don't want that headache right now.

It comes "for free" with the Astrowidgets inheritance, but if this bothers you, I can remove it.

Reply:
Ah, I see. I don't think it needs to be removed then, but it would be nice to have a comment saying that it's there to help with debugging and isn't actually called internally.

Reply:
I could see this being useful eventually, why not 🤷

Reply:
Comment added.